luajit.krun

```diff
@@ -69,6 +74,7 @@ BENCHMARKS = {
 # list of "bench:vm:variant"
 SKIP = [
     "capnproto_decode:GC64:*", # Sometime crashes with GC64
+    "capnproto_decode:RaptorJIT:*"
```

Comment:
Does this crash too?

Reply:
It seems like it even though I'm not building it as GC64

Reply:
What sort of crash are you seeing? We should probably report that upstream to raptorjit.

Reply:
If it's crashing, let's add a comment saying that.